app.js

```diff
@@ -11,19 +11,31 @@ var logger = require('morgan');
 var bodyParser = require('body-parser');
 var mongoose = require('mongoose');
 var router = require('./routes/router');
+var scr = require('./controllers/repositories/systemConfigRepository');
```

Comment:
Strange Path here. Normally, "repository" is not the thing belongs to "controller". Better to have separate folder for repository.